actionview/lib/action_view/render_parser.rb

```diff
@@ -0,0 +1,184 @@
+# frozen_string_literal: true
+
+require "action_view/ripper_ast_parser"
+
+module ActionView
+  class RenderParser # :nodoc:
+    def initialize(name, code, parser: RipperASTParser)
```

Comment:
```suggestion
    def initialize(name, code)
```

There is no other parser being passed, so why this argument?